CONVERSION_GUIDE.md

```
@@ -0,0 +1,148 @@
+  - expression: "variables.containers.all(c, ...)"
+```
+
+C-0198 and C-0210 are the worked examples. The newer style is what to use for new controls. A bug in the container walk lives in one place instead of three.
+
+If you are migrating an existing control that uses the older copy-paste style and you are already in the file for a Rego-to-CEL migration, convert it to the `variables:` style in the same PR. Do not open a sweep PR that touches every control at once.
+
+### Don't forget init and ephemeral containers
+
+A Pod's security posture includes its init containers and ephemeral containers, not just the main containers list. An attacker who can run an init container with `NET_RAW` has the same surface as one who can run a main container with `NET_RAW`. Your container walk should include all three lists:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)